plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -3286,7 +3286,7 @@ if( options.count(name) ) { \
             get_table_rows_result oracle_result =
                     get_table_rows_ex<key_value_index>(fio_table_row_params, oracle_abi);
 
-            FIO_404_ASSERT(!oracle_result.rows.size < 3, "Not Enough Registered Oracles", fioio::ErrorPubAddressNotFound);
+            FIO_404_ASSERT(!oracle_result.rows.size() < 3, "Not Enough Registered Oracles", fioio::ErrorPubAddressNotFound);
```

Comment:
The fee records need to be checked -- if one of the oracles has not called setoraclefees, the chain still crashes. However this check now displays the correct message when no oracles are registered.